### PR TITLE
DIT-620 WIP Add Formatters and fix a few tests

### PR DESCRIPTION
**Pull request by ed0906**

### CODE REVIEW

**Comment by ed0906** [comment]
This is *NOT* ready for merging

File: app/uk/gov/hmrc/bindingtariffclassification/model/MongoFormatters.scala [hunk 1]
```diff
@@ -19,10 +19,26 @@ package uk.gov.hmrc.bindingtariffclassification.model
 import java.time.{Instant, ZoneId, ZonedDateTime}
 
 import play.api.libs.json._
+import uk.gov.hmrc.crypto.{CompositeSymmetricCrypto, Crypted, PlainText}
 import uk.gov.hmrc.play.json.Union
 
 object MongoFormatters {
 
+  private val crypto = CompositeSymmetricCrypto.aes("MZygpewJsCpRrfOrr77j53", Seq.empty) // TODO Pull the real key from config
```
Comment (ed0906) [inline app/uk/gov/hmrc/bindingtariffclassification/model/MongoFormatters.scala]: Not sure the best way to pull this from config given MongoFormatters is an Object

Comment (gokyo) [reply app/uk/gov/hmrc/bindingtariffclassification/model/MongoFormatters.scala]: You can read the config in a new trait or singleton class you create. 
Your object can extend it.

File: app/uk/gov/hmrc/bindingtariffclassification/model/MongoFormatters.scala [hunk 2]
```diff
@@ -19,10 +19,26 @@ package uk.gov.hmrc.bindingtariffclassification.model
...
+
+  private def encrypting(str: String): String = {
+    new String(crypto.encrypt(PlainText(str)).toBase64)
+  }
+
+  private def decrypting(str: String): String = {
+    crypto.decrypt(Crypted.fromBase64(str)).value
+  }
+
+  implicit val stringFormat: Format[String] = Format[String](
```
Comment (gokyo) [inline app/uk/gov/hmrc/bindingtariffclassification/model/MongoFormatters.scala]: I would add the word `crypto` in the variable name

Comment (ed0906) [reply app/uk/gov/hmrc/bindingtariffclassification/model/MongoFormatters.scala]: Done

**Pull request closed without merging**